.github/workflows/handler-test.yml

```diff
@@ -370,12 +370,19 @@ jobs:
           terraform output -no-color -raw proxy_instance_zone
 
       - name: Set up Cloud SDK
-        uses: google-github-actions/setup-gcloud@v0.2.1
+        uses: google-github-actions/setup-gcloud@v0
         with:
           project_id: ${{ secrets.GOOGLE_PROJECT }}
           service_account_key: ${{ secrets.GCP_SA_KEY }}
           export_default_credentials: true
 
+      - name: increasing_the_tcp_upload_bandwidth
+        id: increasing_the_tcp_upload_bandwidth
```

Comment:
I notice here that the other `name` arguments are regular, readable names, but this is snake-cased. Can you make it uniform with the others? 

Also, the ID uses underscores instead of dashes like the others. Is there a benefit to diverging from the rest of the config?

Reply:
done!